#!/usr/bin/env python
PACKAGE = "dynamic_parameter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
 
gen.add("T2S1arm", double_t, 0, "A double parameter",    0.20, -1,   1)
gen.add("T2S2arm", double_t, 0, "A double parameter",    -0.02, -1,   1)
gen.add("T2S3arm", double_t, 0, "A double parameter",    0.25, -1,   1)
gen.add("T3S1arm", double_t, 0, "A double parameter",    0.10, -1,   1)
gen.add("T3S2arm", double_t, 0, "A double parameter",    -0.06, -1,   1)
gen.add("T3S3arm", double_t, 0, "A double parameter",    0.15, -1,   1)


exit(gen.generate(PACKAGE, "dynamic_parameter", "PickPlaceArm"))